custom/panel_templates/Default/assets/js/dataTables.bootstrap4.min.js

```diff
@@ -1,3 +1,4 @@
+// @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt Expat/MIT
```

Comment:
Thanks for this! Are you sure this is datatables version >1.10? Apparently older versions are licensed under GPL/BSD. All other licenses look fine.

Reply:
After some digging it appears the version of dataTables.bootstrap4.min.js is 1.10.19 as this [file hosted on their CDN](https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js) is identical to this one without any changes